packages/core/src/Cardano/types/Genesis.ts

```diff
@@ -13,7 +14,7 @@ export interface CompactGenesis {
   epochLength: number;
   slotsPerKesPeriod: number;
   maxKesEvolutions: number;
-  slotLength: number;
+  slotLength: Seconds;
```

Comment:
Here `CompactGenesis.slotLength: Seconds`, however `EraSummary.slotLenght: Milliseconds` , is it feasible their types to be aligned and `slotLenght` to have one type?

Reply:
You could discard my question since there was already a discussion for it in slack :)